adminSiteClient/EditorCustomizeTab.tsx

```diff
@@ -309,6 +310,26 @@ export class EditorCustomizeTab extends React.Component<{
                                         }
                                     />
                                 </FieldsRow>
+                                <FieldsRow>
+                                    <Toggle
+                                        label={`Facets have uniform y-axis`}
+                                        value={
+                                            // default is absolute
+                                            [
+                                                FacetScaleMode.absolute,
+                                                undefined,
+                                            ].includes(
+                                                yAxisConfig.facetScaleMode
+                                            )
```

Comment:
Is it possible to have this "absolute is default" logic somewhere more "central"? Potentially like how `fontSize` is defined on the axis: https://github.com/owid/owid-grapher/blob/9bc2033bbdc0e834d0d8589a080232d602e6f84e/grapher/axis/AxisConfig.ts#L61-L63

Reply:
Moved it to `AxisConfig` 👍